File: src/main/java/org/profit/app/schedule/job/AnalyticJob.java
package org.profit.app.schedule.job;

import org.profit.app.realm.StockRobot;
import org.quartz.Job;
import org.quartz.JobExecutionContext;
import org.quartz.JobExecutionException;

/**
 * @author TangYing
 *
 * 单线程循环下载
 */
public class AnalyticJob implements Job {

    @Override
    public void execute(JobExecutionContext context) throws JobExecutionException {
        StockRobot.run();
    }
}

File: src/main/java/org/profit/app/schedule/job/AnalyticJob.kt
package org.profit.app.schedule.job

import org.profit.app.StockHall
import org.quartz.Job
import org.quartz.JobExecutionContext
import org.quartz.JobExecutionException

class AnalyticJob : Job {

    @Throws(JobExecutionException::class)
    override fun execute(context: JobExecutionContext) {
        StockHall.analyse()
    }
}